src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukHeader.scala.html

```diff
@@ -79,8 +79,8 @@
           }
 
           @navigation.map { navigation =>
-            <nav aria-label="@params.navigationLabel.getOrElse("Menu")" class="govuk-header__navigation @navigationClasses">
-              <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="@params.menuButtonLabel.getOrElse("Show or hide menu")" hidden>Menu</button>
+            <nav aria-label="@params.navigationLabel.getOrElse(menuButtonText.getOrElse("Menu"))" class="govuk-header__navigation @navigationClasses">
```

Comment:
I done the work here already but I can always strip it out as we have a ticket crated for this work
this was in the ticket https://jira.tools.tax.service.gov.uk/browse/PLATUI-2116
It just felt like a trivial change.